### ISSUE TITLE
Multiple charts performance issue

### ISSUE THREAD

**Issue by manusabarros** [highcharts/highcharts-react#333]
Hi! I have to draw a table of 30 rows and every row has 13 cells, 4 of them with a line chart and 1 with a column chart, so I have to render 150 charts on the page, and the performance is really bad. I'm lazy loading the wrapper component so the table doesn't wait for that cells to be rendered, but it takes about 7 seconds to draw all charts and during that time you can't interact with the page, because the rendering is being blocked.

The charts' options are very simple, I only need the line on the line chart and the bars on the column one, so for example the options for the line chart are the following:

```
options: {
    chart: {
        margin: 0,
        backgroundColor: "transparent",
    },
    series: [
        {
            lineWidth: 2,
            type: "spline",
            color: "#31ABA6",
            data: props.data,
            showInLegend: false,
            marker: { enabled: false },
        },
    ],
    tooltip: { enabled: false },
    xAxis: { visible: false },
    yAxis: { visible: false },
    title: { text: undefined },
    credits: { enabled: false },
    plotOptions: { series: { animation: false } },
}
```

Any ideas on how to make the charts load faster, or else load in the background but allow the user to interact in the meantime? Just in case, the app is built in Next.js and the lazy loading is done with the provided `next/dynamic` feature. Thanks in advance!

**Comment by ppotaczek** [comment]
Hi @manusabarros,

Thank you for contacting us!

The process of creating 150 charts shouldn't take so much time. I have created an example with that amount and it takes about 2 seconds.

Example: https://codesandbox.io/s/highcharts-react-demo-fork-27upm?file=/demo.jsx

There is also an official example with using `setTimeout` for optimization. Please check it: https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/demo/sparkline

Best regards!

**Comment by manusabarros** [comment]
Hi @ppotaczek ,

Thanks for you reply. I found something interesting. In order to make the charts fully responsive, we use the following css:

`
> div[data-highcharts-chart] {
    width: 100% !important;
    height: 100% !important;
}
`

And with the chrome profiling tool I found that using those 2 lines of css the rendering takes almost double of time to show the charts.

Why is that? And is there another way to make the charts fit their containers?
